augment.py

```diff
@@ -0,0 +1,33 @@
+import torch
+import torchvision.transforms as transforms
+from PIL import Image
+#import imgaug.augmenters as iaa
+import os
+
+cur_path = os.path.abspath(os.getcwd())
+
+# Randomly augments the image to change a combination 
+# of the brightness, contrast, saturation, and hue
+# https://pytorch.org/vision/main/generated/torchvision.transforms.ColorJitter.html
+
+def augment(img):
+    transform = transforms.ColorJitter(brightness=.5, contrast=.5, saturation=.5, hue=.30)
+    img = transform(img)
```

Comment:
I'm not sure if this works yet because I haven't gotten the code running, but if it does then I'm surprised that you can pass a PIL Image object to `transforms.ColorJitter()`. It might be good to do:
 `transform = transforms.Compose([ transforms.ToTensor(), transforms.ColorJitter(brightness=.5, contrast=.5, saturation=.5, hue=.30), transforms.ToPILImage() ])`
The idea here is that you start with a PIL Image, then you convert to a tensor for the color jitter, then convert back to a PIL Image to be able to save the augmented image later on line 33.